nix/flakebox.nix

```diff
@@ -318,6 +319,15 @@ rec {
     doInstallCargoArtifacts = false;
   };
 
+  workspaceCargoSort = craneLib.mkCargoDerivation {
```

Comment:
Instead of Nix derivation for it, let's make a flakebox git precommit hook. cherry-pick: https://github.com/fedimint/fedimint/pull/5659